changelog/1.7.0.md

```diff
@@ -0,0 +1,12 @@
+# Scala.meta 1.7.0
+
+
+## Upgrade guide
+
+Contrib:
+- `equal` has been renamed to `isEqual` to avoid accidentally using `equals`,
+  which always compiles but does the wrong thing at runtime.
+- `isEqual` defaults to structual equality, so it's not longer necessary to
+  write `[Structurally]` in `a.isEqual[Structurally](b)`.
+- `(a: Subtype).isEqual(b: Supertype)` will no longer compile, either upcast
```

Comment:
This is not the only case.

For instance in the case of `Term.Name` and `Type.Name` they are sibling types. Thus switching the order is irrelevant most of the time. This is a typesafe equals. Meaning that the rhs must be the lhs type or a subtype of the lhs type.